custom/icds_reports/templates/icds_reports/icds_app/reassignment-message.html

```diff
@@ -0,0 +1,3 @@
+<div class="alert alert-info">
+  The location you have chosen has been deprecated. It has been replaced with {$ selectedLocation.deprecated_to.join(' & ') $}.
```

Comment:
I think we should not use a new term like `replaced` and rather use merge/split/extract/move or just `deprecate`. Plus replaced would be confusing for some operation which does not really replace a location but get extracted from it like in A -> A + B

Reply:
@calellowitz have you considered this yet? I did comment this on the html but the idea was overall even in code.

Reply:
Any conversation about language in these messages probably needs to be resolved on https://docs.google.com/document/d/1LO9oO3Z-JQbeTMr-76sgyVDMFfYWM5eItEvTxJq2E2Y/edit with Nisha and Nitish. That language is mostly their language, but I will flag that they are explicitly not in favor of the word `deprecate` which is why it was removed in the last commit.